lib/src/components/_stacks-tables.less

```diff
@@ -71,6 +71,11 @@
     tbody + tbody {
         border-top: @su2 solid @black-100;
     }
+
+    //  The .user-info include has some padding we don't want when used in tables.
+    .user-info {
```

Comment:
I don't think this belongs here, especially since Stacks doesn't provide `user-info`. I think this would be better suited for a `p0` elsewhere on Core. 🤔

Reply:
`<div class="user-info"></div>` is a shared component that we use in a lot of places. In most cases, it makes sense to have the padding:
<img width="222" alt="screen shot 2018-09-26 at 1 00 54 pm" src="https://user-images.githubusercontent.com/1172461/46096088-40b53600-c18c-11e8-9ae9-1db2f15b9115.png">

I don't think atomic CSS is possible here. I've run into this a few times now.

Reply:
I think we should add a parameter to the shared component that accepts additional classes. In the context of tables, it'd make sense to pass `.p0`. I don't think this CSS belongs in Stacks.